[2024-04-16T10:36:53.688+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-16T10:36:54.236+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: priority_one_number_two_dag_v1.check_sum_number scheduled__2024-02-28T00:00:00+00:00 [queued]>
[2024-04-16T10:36:54.368+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: priority_one_number_two_dag_v1.check_sum_number scheduled__2024-02-28T00:00:00+00:00 [queued]>
[2024-04-16T10:36:54.370+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 6
[2024-04-16T10:36:54.502+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): check_sum_number> on 2024-02-28 00:00:00+00:00
[2024-04-16T10:36:54.529+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=539) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-16T10:36:54.550+0000] {standard_task_runner.py:63} INFO - Started process 545 to run task
[2024-04-16T10:36:54.549+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'priority_one_number_two_dag_v1', 'check_sum_number', 'scheduled__2024-02-28T00:00:00+00:00', '--job-id', '312', '--raw', '--subdir', 'DAGS_FOLDER/priority_one_number_two_dag.py', '--cfg-path', '/tmp/tmpw4ixdgie']
[2024-04-16T10:36:54.557+0000] {standard_task_runner.py:91} INFO - Job 312: Subtask check_sum_number
[2024-04-16T10:36:54.763+0000] {task_command.py:426} INFO - Running <TaskInstance: priority_one_number_two_dag_v1.check_sum_number scheduled__2024-02-28T00:00:00+00:00 [running]> on host eaff2d74d54d
[2024-04-16T10:36:55.131+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ika' AIRFLOW_CTX_DAG_ID='priority_one_number_two_dag_v1' AIRFLOW_CTX_TASK_ID='check_sum_number' AIRFLOW_CTX_EXECUTION_DATE='2024-02-28T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-28T00:00:00+00:00'
[2024-04-16T10:36:55.134+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-16T10:36:55.259+0000] {logging_mixin.py:188} INFO - Total dari list number random : 588
[2024-04-16T10:36:55.261+0000] {logging_mixin.py:188} INFO - Even Sum
[2024-04-16T10:36:55.263+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-04-16T10:36:55.264+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-16T10:36:55.286+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=priority_one_number_two_dag_v1, task_id=check_sum_number, execution_date=20240228T000000, start_date=20240416T103654, end_date=20240416T103655
[2024-04-16T10:36:55.350+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-16T10:36:55.388+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-16T10:36:55.393+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
